cmd/planetmint-god/cmd/initialize_tw.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/go-bip39"
+	"github.com/planetmint/planetmint-go/lib/trustwallet"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagSerialPort = "serial-port"
+)
+
+func InitializeTrustWalletCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "initialize-trust-wallet [mnemonic_word_1][,[mnemonic_word_2],...[mnemonic_word_12],...[mnemonic_word_24]]",
```

Comment:
looking at the other commands I personally would go with `init-tw`, but this is just my personal taste:
```
Available Commands:
  add-genesis-account Add a genesis account to genesis.json
  collect-gentxs      Collect genesis txs and output a genesis.json file
  config              Create or query an application CLI configuration file
  debug               Tool for helping with debugging your application
  export              Export state to JSON
  gentx               Generate a genesis tx carrying a self delegation
  help                Help about any command
  init                Initialize private validator, p2p, genesis, and application configuration files
  keys                Manage your application's keys
  migrate             Migrate genesis to a specified target version
  query               Querying subcommands
  rollback            rollback cosmos-sdk and tendermint state by one height
  start               Run the full node
  status              Query remote node for status
  tendermint          Tendermint subcommands
  tx                  Transactions subcommands
  validate-genesis    validates the genesis file at the default location or at the location passed as an arg
  version             Print the application binary version information
```